Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -2047,6 +2137,310 @@ def unstuck_fetch_stream_command():
+        resourceIDs=resource_ids, region=region, id=container_ids, profileId=profile_id, image=image_name, imageId=image_id,
+        hostname=hostname, complianceIDs=compliance_ids, agentless=agentless, search=search
+    )
+
+    if container_scan_results := client.get_container_scan_results(params=params):
+        table = tableToMarkdown(
+            name="CI Scan Information",
+            t=[
+                {
+                    "ID": container.get("_id"),
+                    "Hostname": container.get("hostname"),
+                    "Scan Time": container.get("scanTime"),
+                    "Image ID": container.get("info", {}).get("imageID"),
+                    "Image Name": container.get("info", {}).get("imageName"),
+                    "Name": container.get("info", {}).get("name"),
+                    "App": container.get("info", {}).get("app"),
+                } for container in container_scan_results
+            ],
+            headers=["ID", "Hostname", "Scan Time", "Image ID", "Image Name", "Name", "App"],
+            removeNull=True,
+        )
+    else:
+        table = "No results found."
+
+    return CommandResults(
+        outputs_prefix="PrismaCloudCompute.ContainersScanResults",
+        outputs_key_field="_id",
+        outputs=container_scan_results,
+        readable_output=table,
+        raw_response=container_scan_results
+    )
+
+
+def get_hosts_info(client: PrismaCloudComputeClient, args: dict) -> CommandResults:
+    """
+    Retrieve a list of hosts information.
+    Implement the command 'prisma-cloud-compute-hosts-info'.
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-hosts-info command arguments.
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    limit, offset = parse_limit_and_offset_values(
+        limit=args.get("limit", "50"), offset=args.get("offset", "0")
+    )
+    collections, account_ids, clusters, resource_ids, region, hostname, compliance_ids, agentless, search = (
+        argToList(args.get("collections")), argToList(args.get("account_ids")), argToList(args.get("clusters")),
+        argToList(args.get("resource_ids")), argToList(args.get("region")), argToList(args.get("hostname")),
+        argToList(args.get("compliance_ids")), args.get("agentless"), args.get("search")
+    )
+    params = assign_params(
+        offset=offset, limit=limit, collections=collections, accountIDs=account_ids, clusters=clusters, resourceIDs=resource_ids,
+        region=region, hostname=hostname, complianceIDs=compliance_ids, agentless=agentless, search=search
+    )
+
+    if hosts_info := client.get_hosts_info(params=params):
+        table = tableToMarkdown(
+            name="Hosts Information",
+            t=[
+                {
+                    "ID": host.get("_id"),
```

Comment:
this is a general note:
I think ID is mandatory in this response, and probably other fields.
in such cases, in general, we should not use get(), because we should fail if it is missing.